Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use a lightweight Python base image
```

Comment:
let's remove this for now. we can migrate essential code from the other repos at a later date.